src/translations/fr/dashboard.json

```diff
@@ -13,6 +13,7 @@
   "latestversionscommits.buttonblock.label": "Explorer",
   "latestversionscommits.buttonblock.link.title": "Explorer les versions de documents",
   "latestversionscommits.buttonblock.title": "Nous avons enregistré {{count}} versions de documents",
+  "latestversionscommits.buttonblock.title_plural": "__STRING_NOT_TRANSLATED__",
```

Comment:
Why is this added in this PR? This should in any case be translated.

Reply:
Fixed here https://github.com/ambanum/OpenTermsArchive.org/pull/131/commits/e5ba03ea32c26116cf50577b72027600ac7a799b